app/uk/gov/hmrc/helptosavefrontend/controllers/IvController.scala

```diff
@@ -40,7 +40,7 @@ class IvController @Inject()(val sessionCacheConnector: SessionCacheConnector,
                              frontendAuthConnector: FrontendAuthConnector)
   extends HelpToSaveAuth(app, frontendAuthConnector) with I18nSupport with Logging {
 
-  def journeyResult: Action[AnyContent] = authorisedForHts { // scalastyle:ignore cyclomatic.complexity method.length
+  def journeyResult(continueURL: String): Action[AnyContent] = authorisedForHts { // scalastyle:ignore cyclomatic.complexity method.length
```

Comment:
it's here - I put it as part of the callback url: https://github.com/hmrc/help-to-save-frontend/blob/master/app/uk/gov/hmrc/helptosavefrontend/controllers/HelpToSaveAuth.scala#L99